src/core/cgroup.c

```diff
@@ -2476,9 +2492,6 @@ static int unit_get_cpu_usage_raw(Unit *u, nsec_t *ret) {
                 _cleanup_free_ char *val = NULL;
                 uint64_t us;
 
-                if ((u->cgroup_realized_mask & CGROUP_MASK_CPU) == 0)
-                        return -ENODATA;
-
```

Comment:
i think this test should stay, but be bypassed if the version check holds (which is cheap if it becomes its own function as suggested above)